packages/react-native-renderer/src/ReactFabricHostConfig.js

```diff
@@ -222,8 +226,8 @@ export function getChildHostContext(
 }
 
 export function getPublicInstance(instance: Instance): null | PublicInstance {
-  if (instance.publicInstance != null) {
-    return instance.publicInstance;
+  if (instance.canonical != null && instance.canonical.publicInstance != null) {
```

Comment:
Why do we need a null-check here? The types claim these are all non-nullable.

Reply:
I don't remember why I did the original check (should've added a comment) but there's some code like this for compatibility with Paper, so types don't match 1:1. I'll double check.